src/esphome/light/light_state.cpp

```diff
@@ -63,6 +66,8 @@ LightColorValues LightState::get_current_values() {
       this->remote_values_ = this->values_ = this->transformer_->get_end_values();
       this->transformer_ = nullptr;
       this->send_values();
+	  if (this->active_effect_ != nullptr)
+		  this->active_effect_->start_internal();
```

Comment:
Revert this, effects are started in `start_effect`

Reply:
This is also to allow flashes while an effect is running.